<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartViewSettings</class>
 <widget class="QDialog" name="ChartViewSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>215</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Impostazioni Grafici</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/images/icons/chart.png</normaloff>:/images/icons/chart.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Finestre Grafici</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="windowsCombo"/>
            </item>
            <item>
             <widget class="QLineEdit" name="newWindowName"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="addWindowButton">
                <property name="text">
                 <string>Aggiungi</string>
                </property>
                <property name="icon">
                 <iconset resource="../mapgraphics/resources.qrc">
                  <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeWindowButton">
                <property name="text">
                 <string>Rimuovi</string>
                </property>
                <property name="icon">
                 <iconset resource="../mapgraphics/resources.qrc">
                  <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Grafico</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="addChartButton">
               <property name="text">
                <string>Aggiungi</string>
               </property>
               <property name="icon">
                <iconset resource="../mapgraphics/resources.qrc">
                 <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="removeChartButton">
               <property name="text">
                <string>Rimuovi</string>
               </property>
               <property name="icon">
                <iconset resource="../mapgraphics/resources.qrc">
                 <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="table">
       <property name="styleSheet">
        <string notr="true">alternate-background-color: #353535;</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mapgraphics/resources.qrc"/>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
